.github/workflows/release.yml

```diff
@@ -0,0 +1,18 @@
+name: Release-Pipeline
+
+on:
+  workflow_run:
+    workflows: [CI-Pipeline]
+    branches: [main]
+    types:
+      - completed
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  call-release-pipeline:
+    if: ${{ github.event_name == 'workflow_dispatch' || (github.event_name == 'workflow_run' && github.event.workflow_run.conclusion == 'success') }}
+    # TODO: Lock this into a release version and create renovate rule to bump it
+    uses: terraform-ibm-modules/common-pipeline-assets/.github/workflows/release.yml@main
```

Comment:
This needs to point to common-release.yml in this repo. And since it is already in this repo I think it an be referenced with a regular path instead of git path, I think it is not relative but always starts at root of repo, so would look like: `./.github/workflows/common-release.yml`